src/components/Home.js

```diff
@@ -0,0 +1,27 @@
+import style from '../styles/home.css';
+
+const Home = () => (
+  <section className={style.section}>
+    <h4>Welcome to our page</h4>
```

Comment:
- [ ] I suggest you increase the font size a little to match the provided wireframe.👍